ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/strategy/transferstrategy/RangeHelper.java

```diff
@@ -22,10 +22,14 @@
 import com.spectralogic.ds3client.models.common.Range;
 import com.spectralogic.ds3client.utils.Guard;
 
+/**
+ * A class used in calculating the remaining data offset(s) and length(s) when we transfer less than an entire
+ * blob during a get.
+ */
 final class RangeHelper {
     private RangeHelper() {}
 
-    static ImmutableCollection<Range> replaceRange(final ImmutableCollection<Range> existingRanges,
+    protected static ImmutableCollection<Range> replaceRange(final ImmutableCollection<Range> existingRanges,
```

Comment:
The codacity thing complained about using default visibility.  It quit complaining after I set the 
method protected.